docs/apps/lnw/es2k/es2k-linux-networking.md

```diff
@@ -16,6 +16,68 @@ To enable this feature we have,
 - `p4rt-ctl`: This python CLI includes a p4runtime client. Programs IPU E2100 with runtime rules by communicating with gRPC server.
 - `Kernel stack`: All underlay related configurations are picked by `kernel monitor` thread via netlink events in `infrap4d` and these are programmed in IPU E2100 by calling TDI front end APIs.
 
+### P4 program changes in 24.01
+
+Modify p4 programm with below changes, to enable support for
+
+- IPv6 traffic with VxLAN support.
+- VxLAN traffic fails when remote overlay VMs MAC is learnt on OvS, before DUTs overlay VMs MAC address.
```

Comment:
It's not clear what are we enabling the support for here. The statement describes the problem but not the functionality. 1st one sounds like feature enhancement but  wasn't IPv6 code already available. Are both bug fixes?

Reply:
IPv6 is already available but due to a defect in p4 program, without this p4 programs change, IPv6 with VxLAN traffic will fail.